documentation/01-Prerequisites.md

```diff
@@ -6,76 +6,46 @@ These are one-time setup instructions that should be executed prior to following
 
 - [Prerequisites for building and using MLOS](#prerequisites-for-building-and-using-mlos)
   - [Contents](#contents)
-  - [Requirements](#requirements)
-  - [Clone the repository](#clone-the-repository)
-  - [Install build tools](#install-build-tools)
-    - [Docker](#docker)
+  
+  - [Linux](#linux)

```

Comment:
Should we add an OS X section as well, maybe just for Python and Docker?

Reply:
If we decide to add description for OS X then may be we should split the guide into separate files (one file for each OS) to keep each guide compact and focused? What do you think?

Reply:
Probably makes sense.  There's some redundancy then but so it goes.
One complication we're still working out is how best to keep relative links in the markdown files working so that browsing on github is natural (e.g. stays within tree, renders, etc.) but the github.io pages always take you back to master.
I think I have a hack to handle it.  Will work on it tomorrow.

Reply:
I was about to start the OS X section, but shy away as I suspected I was not going to be able to finish it. Agreed on splitting if we add more than 2 OS